apis/python/src/tiledbsoma/io.py

```diff
@@ -332,7 +332,7 @@ def create_from_matrix(
         platform_config=platform_config,
     )
 
-    if soma_ndarray.soma_type == "SOMADenseNDArray":
+    if isinstance(soma_ndarray, DenseNDArray):
```

Comment:
For a different change, but this seems like it would be better served by putting a `_write_matrix(self, src_matrix)` method on each of `SparseNDArray` and `DenseNDArray`

Reply:
@thetorpedodog please file a GH issue! :)